lib/python/vdm/tests/server/deployment_test.py

```diff
@@ -87,12 +88,13 @@ def test_get_deployment(self):
         """
         ensure GET Deployment
         """
+        headers = {'Accept': 'application/json'}
```

Comment:
Its related to the deployment json.  Actually, Deployment API (databases/{database_id}}/deployment/) gives the deployment information in both json as well as XML format. As we had made some changes in it for download configuration, we had to make changes on above test case which is related to deployment information in json.

Also, we have already written server side test cases related to deployment XML API. So we think that it might be sufficient for the download.